xmake/rules/python/cython/xmake.lua

```diff
@@ -0,0 +1,63 @@
+--!A cross-platform build utility based on Lua
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+-- Copyright (C) 2015-present, TBOOX Open Source Group.
+--
+-- @author      Wu, Zhenyu
+-- @file        xmake.lua
+--
+
+rule("python.cython")
+    set_extensions(".py", ".pyx")
+
+    on_load(function (target)
+        local language = target:extraconf("rules", "python.cython", "language")
+        if language == "c" then
+            target:add("deps", "c")
```

Comment:
Did you test it? we cannot add c/c++ deps to target. it's rule, not target.

we just add `add_deps("c", "c++")` in rule scope. and need not extra language configuration.